src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -176,6 +177,12 @@ function MoneyRequestView({report, betas, parentReport, policyCategories, should
                         </View>
                     </OfflineWithFeedback>
                 )}
+                {!hasReceipt && canEdit && !isSettled && Permissions.canUseViolations() && (
```

Comment:
This caused regressions:
- #31407
- #31418

`Permissions` is not imported. Anyone please fix these deploy blockers?